azerothcore/azerothcore-wotlk
# fix(Core/Database): fix wrong Field::IsNull() return value on null binary fields

## Changes Proposed:
- Changes `Field::IsNull()` so that it returns `true` on binary fields if the length of the data is 0

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/5970

## SOURCE:


## Tests Performed:

- Logged in on my account and moved around a bit in the world

## How to Test the Changes:

Nothing specific to do, make sure you are able to log in.
Also try to log in on an account with 2FA enabled, if possible.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
